Changes.md

```diff
@@ -1,3 +1,22 @@
+### Beta 10.2 (2/26/18):
```

Comment:
this needs to be added to b10 branch as well https://github.com/akotlar/bystro/issues/74